typings/index.d.ts

```diff
@@ -75,7 +75,7 @@ declare module 'canvas-constructor' {
 		public isPointInPath(dx: number, dy: number, fillRule: FillRuleType): boolean;
 		public isPointInStroke(dx: number, dy: number): boolean;
 		public lineTo(dx: number, dy: number): this;
-		public measureText(text: string): number;
+		public measureText(text: string): MeasureTextProps;
```

Comment:
The type is called `TextMetrics`, you can see the line below using it.
```suggestion
		public measureText(text: string): TextMetrics;
```